src/uu/rm/src/rm.rs

```diff
@@ -115,11 +115,20 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
             verbose: matches.get_flag(OPT_VERBOSE),
         };
         if options.interactive == InteractiveMode::Once && (options.recursive || files.len() > 3) {
-            let msg = if options.recursive {
-                "Remove all arguments recursively?"
-            } else {
-                "Remove all arguments?"
-            };
+            let msg: String = format!(
+                "remove {} {}{}",
+                files.len(),
+                if files.len() > 1 {
+                    "arguments"
+                } else {
+                    "argument"
+                },
+                if options.recursive {
+                    " recursively?"
+                } else {
+                    "?"
+                }
+            );
```

Comment:
About about?

```suggestion
            let message = format!(
                "Are you sure you want to remove {} {}{}",
                files.len(),
                if files.len() > 1 { "arguments" } else { "argument" },
                if options.recursive { " recursively?" } else { "?" }
            );
```

Reply:
yes thats a lot cleaner, but the pre commit hook would change it back